config/metadata/basic_metadata.yaml

```diff
@@ -2,71 +2,131 @@ attributes:
   abstract:
     type: string
     multiple: true
-  access_right:
+    index_keys:
```

Comment:
It appears that order has been changed. As I'm not familiar with these files, is order significant for the keys?

Reply:
If no, the order is not significant, I'd prefer to see the keys not re-ordered as it makes the review more complicated. That is not a blocker, but makes the effort of review more significant.

Reply:
i've been trying to keep the key order alphabetical, for ease of reference. it would be good to fix this up to conform to that convention.

Reply:
I altered the order in the pursuit to verify that we had captured all the necessary pieces of info to be indexed. This was purely for cross referencing. I can update this to be alphabetical again. No biggy for me.